astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -980,15 +980,15 @@ def test_fast_tab_with_names(parallel, read_tab):
     read_tab(content, data_start=1, parallel=parallel, names=head)
 
 
-@pytest.mark.skipif(not os.getenv('TEST_READ_HUGE_FILE'),
-                    reason='Environment variable TEST_READ_HUGE_FILE must be '
+@pytest.mark.skipif(not os.getenv('TEST_INTENSIVE'),
+                    reason='Environment variable TEST_INTENSIVE must be '
                     'defined to run this test')
```

Comment:
```suggestion
@pytest.mark.skipif(os.getenv('TEST_INTENSIVE', 0) == 0,
                    reason='Environment variable TEST_INTENSIVE must be defined to run this test')
```
Just found that I intuitively expected the env var set to 0 to _disable_ the tests; this would also be more consistent with how we are using the `ASTROPY_USE_` vars (also noticed that setup is using `os.environ.get()`, whatever the correct Zen is here ;-).
I prefer it without further linebreak, but unfortunately this would make the docs example break the line...

Reply:
I don't think we have any consistent zen in this department:

https://github.com/astropy/astropy/blob/73c57e16014a11de8ffc7ee75b66ba7b54ed51c2/.github/workflows/ci_workflows.yml#L11-L13

I would have named it `SHOW_ME_THE_MONEY` but this works too.

Reply:
But those are checking the value as well.
`TEST_INTENSIVE="False"` or `TEST_INTENSIVE=0` did let the tests run.

Reply:
Re: values -- Yes, I agree with @dhomeier . The check should be more strict to only let it run when the value is 1, as documented.

Reply:
I'd be willing to allow anything that could be interpreted as `True`, but in fact my suggestion will still accept `'false'` as well...